x/gov/keeper/keeper.go

```diff
@@ -66,6 +69,17 @@ func NewKeeper(
 	}
 }
 
+// SetHooks sets the hooks for governance
+func (keeper *Keeper) SetHooks(gh types.GovHooks) *Keeper {
```

Comment:
don't really need to return the keeper here, but it's ok I guess

Reply:
Was just keeping the standard from staking